requirements.txt

```diff
@@ -0,0 +1 @@
+nose==1.3.6
```

Comment:
Yeah, just put nose here for running tests... I guess I could just install it in the `.drone.yml` and the `Dockerfile` for local use. Thoughts?